docs/source/working_with_ontologies.md

```diff
@@ -1,21 +1,142 @@
-# Description of YAML format specification
+# How to work with ontologies
 
-This file describes how you can create ontologies using YAML.
+OSP-core supports ontologies in the following formats:
+- OWL ontologies
+- RDFS vocabularies _(very limited support)_
```

Comment:
I would refer to the explanation below and remove the word "very" --> _(limited support; see below for more information)_

Reply:
:+1:

Reply:
Done